components/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt/src/test/java/org/wso2/carbon/identity/api/resource/mgt/dao/APIResourceManagementDAOImplTest.java

```diff
@@ -82,80 +91,123 @@ public void tearDown() throws Exception {
     @DataProvider
     public Object[][] getAPIResourcesCountData() {
         return new Object[][]{
-                {TENANT_ID, new ArrayList<>(), 2},
-                {INVALID_TENANT_ID, new ArrayList<>(), 0},
+                {TENANT_ID, TENANT_ID, new ArrayList<>(), false, 2},
+                {TENANT_ID, 1001, new ArrayList<>(), true, 2},
```

Comment:
```suggestion
                {TENANT_ID, SUB_ORG_TENANT_ID, new ArrayList<>(), true, 2},
```
let's define a constant and reuse